microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -1,103 +1,110 @@
 
-1. Configure your WebView2 WinUI 2 (UWP) app for remote debugging:
+1.  Configure your WebView2 WinUI 2 (UWP) app for remote debugging:
 
-    To enable remote debugging, two environment variables must be set in your project.
-
-    These variables must be set before the `CoreWebView2` creation before either the `WebView2.Source` property is set or the `WebView2.EnsureCoreWebView2Async` method is called.
+    To enable remote debugging, two environment variables must be set in your project.  These variables must be set before creating the `CoreWebView2` instance, and before either setting the `WebView2.Source` property or calling the `WebView2.EnsureCoreWebView2Async` method.
 
     Set the following variables:
 
-    `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS = --enable-features=msEdgeDevToolsWdpRemoteDebugging`
+    ```
+    "WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging"
+    "WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1"
+    ```
+    
+    For example, if you're using the [WebView2 Sample UWP app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/webview2_sample_uwp), you can set those environment variables by adding the following lines in the `Browser.xaml.cs` file:
     
-    `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE = 1`
+    ```csharp
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging");
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1");
+    ```
+
+    Those are lines 33 and 34 in this screenshot:
+
+    ![Browser Arguments Remote Debugging](./remote-debugging-images/browser-arguments-remote-debugging.png)
 
-    If you are using the [WebView2 Sample UWP app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/webview2_sample_uwp) you will add the following lines (lines 33 and 34) in the `Browser.xaml.cs` file:
+1.  Launch your app.
 
-    ![Browser Arguments Remote Debugging](./media/browser-arguments-remote-debugging.png)
+1.  Go to `http://<Device Portal URL>/msedge`, such as `http://localhost:50080/msedge`, and make sure your WebView2 instance appears in the list:
 
-1. Launch your app.
+    ![Debuggable WebView2 Instance](./remote-debugging-images/debuggable-webviews.png)
 
-1. Navigate to `http://<Device Portal URL>/msedge` and verify your WebView2 instance appears in the list:
+1.  Go to `edge://inspect`.  In the **Connect to a remote Windows device** text box, enter `http://<Device Portal URL>`, such as `http://localhost:50080`, and then click **Connect to Device**.
 
-    ![Debuggable WebView2 Instance](./media/debuggable-webviews.png)
+1.  Verify that you can successfully connect, so that your debuggable WebView2 control, named **Edge**, is listed under your machine name:

```

Comment:
@maurawinstanley - is the wv2 control usually named "Edge" in this UI?  Could there be multiple?